external/llvm-project/mlir/test/Dialect/Tosa/tosa-partition-run.mlir

```diff
@@ -0,0 +1,46 @@
+// RUN: mlir-opt %s --tosa-to-linalg-on-tensors --tosa-to-standard --linalg-detensorize \
```

Comment:
This is a simple arbitrary test of a convolution followed by a RelU, generated from tensorflow with known inputs so we can test both that it got the right answer and that --tosa-partition-pipeline didn't affect the answer.